src/funfuzz/ccoverage/gatherer.py

```diff
@@ -27,6 +27,16 @@
 RUN_COV_LOG = logging.getLogger("funfuzz")
 
 
+def ccov_time():
+    """Specify the amount of time (in seconds) to run the code coverage build. This separate function is needed so
```

Comment:
You can monkeypatch globals. It doesn't need to be a function.

Reply:
Great idea, thanks!